GWAS/PCA.ipynb

```diff
@@ -651,7 +751,9 @@
     "            paste(\"The new sample size after outlier removal is:\",nrow(pc) - nrow(outliers),\"\\n\"))\n",
     "        #\n",
     "        outliers <- outliers %>%\n",
-    "        select(FID,IID)\n",
+    "        mutate(FID = IID) %>% \n",
+    "        select(FID, IID)%>% \n",
+    "        filter(!is.na(IID))\n",
```

Comment:
why would we even have NA in IID? You are not supposed to "fix" a problem like this.